src/main/java/org/fundacionjala/pivotal/ReadProperties.java

```diff
@@ -0,0 +1,47 @@
+package org.fundacionjala.pivotal;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+/**
+ * Class which reads properties file.
+ * @author Kevin Sanchez AWT - [01].
+ * @version 0.1
+ */
+public class ReadProperties {
+    private Properties properties;
+    private InputStream input;
+
+    /**
+     * Constructor which reads the file.
+     */
+    private ReadProperties() {
+        properties = new Properties();
+        input = null;
+        try {
+            input = new FileInputStream("config.properties");
+            properties.load(input);
+        } catch (IOException ex) {
+            ex.printStackTrace();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use a logger to log this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1148)